src/webapp/components/landing-page-edit-dialog/LandingPageEditDialog.tsx

```diff
@@ -27,16 +28,26 @@ const buildDefaultNode = (type: LandingNodeType, parent: string, order: number)
         content: undefined,
         children: [],
         modules: [],
+        permissions: {
+            publicAccess: "r-------",
+            userAccesses: [],
+            userGroupAccesses: [],
+        },
+        executeOnInit: executeOnInit,
```

Comment:
That's weird. It seems we are returning a `LandingNode` (and that's how it's used: `useState<LandingNode>`),  but `executeOnInit`is not a prop of the type. So I don't think this prop can be really being used (at least not in a type-safe way)